docs/en/week09/09-3.md

```diff
@@ -21,16 +21,21 @@ We can express the above architecture mathematically as
 $$
 \textcolor{#a7ca7c}{\vect{h}}=f(\boldsymbol{W_h}\textcolor{#62a2af}{\vect{y}}+\boldsymbol{b_h})\\
 \textcolor{#b46fa4}{\vect{\tilde{y}}}=g(\boldsymbol{W_y}\textcolor{#a7ca7c}{\vect{h}}+\boldsymbol{b_y})\\
+\aqua {\vect{q}} \\
```

Comment:
For the line above, you want to replace `\textcolor{#a7ca7c}{\vect{h}}` with `\vh`.